composer.json

```diff
@@ -0,0 +1,73 @@
+{
+    "name": "blumilksoftware/escooters",
+    "type": "project",
+    "description": "Escooters project.",
+    "keywords": ["framework", "laravel"],
+    "license": "MIT",
+    "require": {
+        "php": "^8.2",
+        "guzzlehttp/guzzle": "^7.7.0",
+        "inertiajs/inertia-laravel": "^0.6.9",
+        "laravel/framework": "^10.12.0",
+        "laravel/sanctum": "^3.2.5",
+        "laravel/tinker": "^2.8.1",
+        "ext-pdo": "*"
+    },
+    "require-dev": {
+        "blumilksoftware/codestyle": "^v2.2.0",
```

Comment:
Why `v` has been added?
This is not needed and in the other packages you don't use `v` prefix.

Reply:
I think it's a typo on my part

Reply:
fixed